packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -177,6 +177,7 @@ class WalletsController {
       const success = WalletsController.service.delete(id)
       if (success) {
         windowManage.broadcast(Channel.Wallets, WalletsMethod.GetAll, WalletsController.getAll())
+        windowManage.broadcast(Channel.Wallets, WalletsMethod.GetActive, WalletsController.getActive())
```

Comment:
Are we 100% sure these two events are broadcast and processed in the exact first `getAll` then `getActive` order?

Reply:
I will try to make them to be one